<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EscapeDialog</class>
 <widget class="QWidget" name="EscapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>711</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="uiresources.qrc">
    <normaloff>:/Resources/Solar Car Team Icon.ico</normaloff>:/Resources/Solar Car Team Icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background: transparent;
font-family: &quot;AvenirLTStd&quot;;
font-size: 17px;</string>
  </property>
  <layout class="QGridLayout" name="escapeLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="4" column="1">
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background: rgb(51,51,51);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="lowerSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="rightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="EscapeWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;AvenirLTStd&quot;;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="escapeTitleBarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
background: url(:/Resources/SubTitleBar.png);
border-bottom: 1px solid rgb(70,70,70);
</string>
        </property>
        <layout class="QGridLayout" name="titleBarLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="exitButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{
background: rgb(70,70,70);
border:1px solid rgb(90,90,90);
border-radius: 5px;
margin: 3px;
}
QToolButton:hover{
background: rgb(80,80,80);
border:1px solid rgb(90,90,90);
border-radius: 5px;
margin: 3px;
}
QToolButton:pressed{
background: rgb(60,60,60);
border:1px solid rgb(90,90,90);
border-radius: 5px;
margin: 3px;
}
</string>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="escapeTitleBarLabel">
           <property name="font">
            <font>
             <family>AvenirLTStd</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 18px;
background: none;</string>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>Menu</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="backgroundWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background: rgb(51,51,51);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="optionslLayout">
           <item>
            <widget class="QListWidget" name="optionsPanelWidget">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>197</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">margin: 3px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="optionsDisplayWidget">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>AvenirLTStd</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="aboutPageWidget">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="placeholderTitleBarLabel">
                 <property name="text">
                  <string>Telemetry GUI</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="placeholderTextBrowser">
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'AvenirLTStd'; font-size:17px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:17px;&quot;&gt;asdfasfasdfasdf&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="settingsPageWidget">
              <property name="styleSheet">
               <string notr="true">QLabel{
padding: 7px;
}
QLineEdit{
margin-right: 250px;
}
QPushButton{
margin-right: 20px;
padding: 3px;
}</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <widget class="QLineEdit" name="baudrateText">
                 <property name="styleSheet">
                  <string notr="true">
background: rgb(70,70,70);</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="multiWindowLabel">
                 <property name="text">
                  <string>Multi-window</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="baudrateLabel">
                 <property name="text">
                  <string>Baudrate</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="portnameText">
                 <property name="styleSheet">
                  <string notr="true">
background: rgb(70,70,70);
</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="fullscreenLabel">
                 <property name="text">
                  <string>Fullscreen</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QCheckBox" name="multiWindowCheckBox">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QCheckBox" name="fullscreenCheckBox">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="logPathLabel">
                 <property name="text">
                  <string>Log Path</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="portnameLabel">
                 <property name="text">
                  <string>Portname</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <layout class="QHBoxLayout" name="fileDialogLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="fileDialogText">
                   <property name="styleSheet">
                    <string notr="true">margin-right: 5px;
padding-right: none;

background: rgb(70,70,70);</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="fileDIalogPushButton">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(60,60,60);
border: 1px solid rgb(20,20,20);
border-radius: 7px;
background-image: url(:/Resources/BrowseButton.png);</string>
                   </property>
                   <property name="text">
                    <string>Browse</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="exitPushButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>AvenirLTStd</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">margin: 2px;
margin-left:670px;
color: rgb(20,20,20);
background-image: url(:/Resources/BrowseButton.png);
border: 2px solid rgb(20,20,20);
border-radius: 7px</string>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="leftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="uiresources.qrc"/>
 </resources>
 <connections/>
</ui>
